### PR TITLE
Add `remove_comment` command to Buildozer.

### PR DESCRIPTION
**Pull request by Zilulil**
This command removes the comment attached to the specified rule, attribute, or value in an attribute.

### CODE REVIEW

File: edit/buildozer.go [hunk 1]
```diff
@@ -348,6 +348,31 @@ func cmdRemove(opts *Options, env CmdEnvironment) (*build.File, error) {
 	return nil, nil
 }
 
+func cmdRemoveComment(opts *Options, env CmdEnvironment) (*build.File, error) {
+	switch len(env.Args) {
+	case 0: // Remove comment attached to rule
+		env.Rule.Call.Comments.Before = nil
+		env.Rule.Call.Comments.Suffix = nil
```
Comment (vladmos) [inline edit/buildozer.go]: A rule (and in theory any node in the AST) may have an after-comment, please nullify `Rule.Call.Comments.After` too.

    my_rule(
        name = "foo" ,
    )
    # this comment is attached to the rule above

File: edit/buildozer.go [hunk 2]
```diff
@@ -348,6 +348,31 @@ func cmdRemove(opts *Options, env CmdEnvironment) (*build.File, error) {
...
+			attr.Comments.Before = nil
+			attr.Comments.Suffix = nil
+			attr.LHS.Comment().Before = nil
+			attr.RHS.Comment().Suffix = nil
+		}
+	case 2: // Remove comment attached to value
+		if attr := env.Rule.Attr(env.Args[0]); attr != nil {
+			if expr := ListFind(attr, env.Args[1], env.Pkg); expr != nil {
+				expr.Comments.Before = nil
+				expr.Comments.Suffix = nil
```
Comment (vladmos) [inline edit/buildozer.go]: The same as above (although arguments normally have just before or suffix comments, it's cleaner to remove everything).

Comment (Zilulil) [reply edit/buildozer.go]: Done, additionally nil'd out every field on the attr and attr.LHS and attr.RHS.

Comment (vladmos) [reply edit/buildozer.go]: Good catch, a comment like ` attr = value  # comment` will actually be attached to the RHS, not to the assignment node.

**Pull request merged**